app/src/main/res/layout/activity_profile_page.xml

```diff
@@ -162,17 +162,12 @@
 
         android:background="@color/white"
         android:orientation="horizontal">
-        <EditText
-            android:id="@+id/info_profile_description"
```

Comment:
Because of this field removed, the ProfilePageActivitityTest failed, the part of the test that used the description can be removed.

Reply:
I will remove it!